src/Compilers/CSharp/Portable/LanguageVersion.cs

```diff
@@ -469,13 +503,13 @@ public static LanguageVersion MapSpecifiedToEffectiveVersion(this LanguageVersio
                 case LanguageVersion.Latest:
                 case LanguageVersion.Default:
                 case LanguageVersion.LatestMajor:
-                    return LanguageVersion.CSharp10;
+                    return LanguageVersion.CSharp11;
                 default:
                     return version;
             }
         }
 
-        internal static LanguageVersion CurrentVersion => LanguageVersion.CSharp10;
+        internal static LanguageVersion CurrentVersion => LanguageVersion.CSharp11;
```

Comment:
This is used in tests only, I think. Can't the tests use `LanguageVersion.Latest.MapSpecifiedToEffectiveVersion()` instead?

This is something to consider separately, not in this PR.

Reply:
Seems a good idea for a separate PR